linode/manager
# change: [M3-7887] - VPC Not Recommended Configuration Tooltip Text Revision

## Description 📝
This PR updates the revised tooltip copy in `SubnetLinodeRow` component. Tooltips that contain anchor tags linking to external documents are not accessible. Hence anchor tag is removed.

## Changes  🔄
List any change relevant to the reviewer.
- Removed anchor tag from tooltip text in `SubnetLinodeRow`
- Removed NETWORK_INTERFACES_GUIDE_URL url since it is not used any more. 

## Target release date 🗓️
10/28

## Preview 📷
| Before  | After   |
| ------- | ------- |
|![image](https://github.com/user-attachments/assets/b6d32f41-0810-44a6-af05-3dc0c05d494a) |![image](https://github.com/user-attachments/assets/5d48cc6a-4d65-44d7-acee-e3d370fd6dce)|

## How to test 🧪

### Verification steps
(How to verify changes)
- Navigate to vpc subnet details page
- Expand subnet with linode that has unrecommended configurations
- Verify the copy and confirm anchor tag removal.
- Confirm no regression. 

## As an Author I have considered 🤔

*Check all that apply*
